lib/active_merchant/billing/gateways/d_local.rb

```diff
@@ -294,6 +294,7 @@ def headers(post, options = {})
           'Authorization' => signature(post, timestamp)
         }
         headers['X-Idempotency-Key'] = options[:idempotency_key] if options[:idempotency_key]
+        headers['X-Dlocal-Payment-Source'] = options[:payment_source] if options[:payment_source]
```

Comment:
Is this static? If so you may want to use the existing pattern that `application_id` offers

Reply:
I'll update it, I new we had a field like that but I could remember the name

Reply:
`application_id` is not a field in option, it's a class attribute. So in the gateway class you can access it by calling `application_id` and then it's set on the class (`ActiveMerchant::Billing::{{gateway}}.application_id = 'xyz'`). 

AuthorizeNet seems to follow a similar pattern to this and uses application_id if you'd like to copy it